packages/core-web/src/index.js

```diff
@@ -24,36 +24,54 @@ function scrollToUrlAnchorHeading() {
 window.onload = scrollToUrlAnchorHeading;
 
 function detectAndApplyHeaderStyles() {
-  const headerSelector = jQuery('header[sticky]');
+  document.querySelectorAll('h1, h2, h3, h4, h5, h6').forEach((heading) => {
+    if (heading.id) {
+      heading.removeAttribute('id');
+    }
+  }); // to avoid duplicated id problem
```

Comment:
I think removing the IDs will prevent us from being able to access these headings by hash links (e.g. index.html#title). :o Is this intended behaviour?

Reply:
Sorry removed in 083cc63! The deleting of id for the headers was removed in #1953